saleor/payment/interface.py

```diff
@@ -60,3 +70,23 @@ class GatewayConfig:
     # Each gateway has different connection data so we are not able to create
     # a unified structure
     connection_params: Dict[str, Any]
+    store_customer: bool = False
+
+
+@dataclass
+class CreditCardInfo:
+    """Uniform way to represent Credit Card information"""
+
+    last4: str
```

Comment:
Not `last_4`? Braintree also seems to write it like that.

Reply:
Changed.